### PR TITLE
Feat/frontend cleanup

### PR DESCRIPTION
**Pull request by Herrekt**
Removed console logs of objects

### CODE REVIEW

File: collaboration-service/index.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ io.on("connection", (socket) => {
 
     // Client joins a specific room.
     socket.on("join", ({ room }) => {
+        // if (io.sockets.adapter.rooms.get(room).size === 1) {
+        //     socket.broadcast.to(`${room}`).emit("update")
+        // }
```
Comment (hiongkaihan) [inline collaboration-service/index.js:35]: Is this suppose to be commented? If not can put it in another PR

**Review by hiongkaihan — CHANGES_REQUESTED** [review]

File: collaboration-service/index.js (hunk 1 above)
Comment (ongweisheng) [reply collaboration-service/index.js:35]: LGTM if this issue is resolved

**Review by ongweisheng — APPROVED** [review]

**Review by hiongkaihan — APPROVED** [review]
LGTM
